scripts/start-configuration

```diff
@@ -160,7 +160,7 @@ fi
 
 : "${MODPACK_PLATFORM:=${MOD_PLATFORM:-}}"
 
-if [[ $MODPACK_PLATFORM -a $TYPE ]]; then 
+if [[ -z $MODPACK_PLATFORM || -z $TYPE ]]; then 
```

Comment:
That looks more confusing now. Sorry for my wrong suggestion initially, but can you use this instead to get back to my original intent:

```suggestion
if [[ $MODPACK_PLATFORM && $TYPE ]]; then 
```